arangod/Graph/Graph.h

```diff
@@ -46,12 +46,23 @@ struct ServerDefaults;
 namespace graph {
 
 class EdgeDefinition {
+ private:
+  enum EdgeDefinitionType {
+    DEFAULT,
+    SMARTTOSMART,
```

Comment:
I would prefer if we can spend some `_`
e.g.
```suggestion
    SMART_TO_SMART,
```

that would make it more readable imo